subsys/net/ip/ipv6_fragment.c

```diff
@@ -472,6 +472,15 @@ enum net_verdict net_ipv6_handle_fragment_hdr(struct net_pkt *pkt,
 	more = flag & 0x01;
 	net_pkt_set_ipv6_fragment_offset(pkt, flag & 0xfff8);
 
+	if (more && net_pkt_get_len(pkt) % 8) {
+		/* Fragment length is not multiple of 8, discard
+		 * the packet and send parameter problem error.
+		 */
+		net_icmpv6_send_error(pkt, NET_ICMPV6_PARAM_PROBLEM,
+				      NET_ICMPV6_PARAM_PROB_HEADER, 0x04);
```

Comment:
The original code had last parameter as 0 but you change it to 0x04, what the value means here? Could you add a comment and/or use a define (if it makes sense) for this magic number.

Reply:
Indeed. 0x04 is the offset of the "Payload Length" field in the IPv6 header. I will add a comment.

Reply:
Maybe better to use a define of some kind rather than a magic number. If a define does not exist, please add it.

Reply:
I added a define in include/net/net_ip.h but I am not 100% sure if this is the right place.